pyo3-ffi/src/cpython/mod.rs

```diff
@@ -21,7 +21,7 @@ pub(crate) mod import;
 pub(crate) mod initconfig;
 // skipped interpreteridobject.h
 pub(crate) mod listobject;
-#[cfg(Py_3_13)]
+#[cfg(all(Py_3_13, not(Py_LIMITED_API)))]
```

Comment:
This is a bug but no one noticed because no one tried to use the bindings on the limited API, I think?

Reply:
ah so it is! Thanks...